scripts/install/installer

```diff
@@ -1293,11 +1295,17 @@ class Cluster(AbstractConfig):
 class PNode(JumpScale7):
     def ipmi(self, *args):
         ipmiip = get_ipaddress(self.node["ipmi"]["ipaddress"])
+        if "controller" in self.node["roles"]:
+            ip_fields = ipmiip.split(".")
```

Comment:
should be something like
network = netaddr.IPNetwork(self.config['networks']['impi']['network'])
ipmiip = str(network.broadcast - self.noe['ipmi']['ip-lsb']